modules/core/client/app/init.js

```diff
@@ -12,7 +12,23 @@ angular.module(ApplicationConfiguration.applicationModuleName).config(['$locatio
   }
 ]);
 
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
+angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, $http, Authentication, CORE_CONST) {
+
+  //set token header
+  var auth_token = localStorage.getItem('auth_token');
```

Comment:
Session cookies cause a lot of problems for Cordova clients, if it is possible at all.  For me, this is the main reason for token based auth.

 https://cordovablogsblogs.wordpress.com/2015/03/08/handling-cookies-in-phonegapcordova/